packages/nulp_elite/src/App.js

```diff
@@ -14,10 +14,9 @@ import { DEFAULT_THEME, initializeI18n,H2, } from "@shiksha/common-lib";
 import { useTranslation } from 'react-i18next'
 import Framework from "pages/Frmework";
 import User from "pages/User";
-import { extendTheme, NativeBaseProvider } from "native-base";
-// import dotenv from "dotenv";
-import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
+import { extendTheme } from "native-base";
```

Comment:
Why it is needed? @snehalsabade

Reply:
We are not using extendTheme so I can remove it.